### DISCUSSION TITLE
file size exceeed? :(

### DISCUSSION THREAD

**Discussion by gerroon** [vrtmrz/obsidian-livesync#370] [Q&A]
Hi

I am using this between Win and IOS. Recently the IOS Obsidian plugin started complaining about sync problems due to file size exceeded, thinking there was an issue with file size, I increased the setting too 1000MB and it still complaints about it. I do not have a file that is 1GB in my small vault. I doubt this is a real fizle size issue,it is probably miscalculating it.

The biggest file in the repo is 40MB.

**Comment by Hugo-Persson** [comment]
I am also getting this warning for some files, what is the limit and can it be tweaked?

**Comment by vrtmrz** [comment]
Maximum file size can be configured in the `Sync settings` -> `Targets` -> `Maximum file size`.
However, It will be very nice to report the actual message.

**Reply by Chlorgas** [reply]
Hi @vrtmrz !
I checked my config with all extra options enabled (advanced features, poweruser features, edge case treatment features) but I couldn't get the setting. Is there any other possible solution for synching bigger files?

**Comment by michaelschapira** [comment]
Hi @vrtmrz - Thank you for all the work you'd doing. 
I finally taken the time to configure your plugin and it really rocks.  THANK YOU. 

I cannot find the setting you're referring to. 

I am facing the same issue. Here is the log generated (sanitized).  

```
<vault>-80b60cf58788db69:2/28/2025, 10:53:22 AM->Replication closed
<vault>-80b60cf58788db69:2/28/2025, 10:53:22 AM->[HiddenFileSync]  Hidden file will skip 0 files
<vault>-80b60cf58788db69:2/28/2025, 10:53:22 AM->P2P is not enabled
<vault>-80b60cf58788db69:2/28/2025, 10:53:22 AM->Cache initialized 300 / 50000000
<vault>-80b60cf58788db69:2/28/2025, 10:53:29 AM->The storage file has been changed but exceeds the maximum size. Skipping: assets/<filename>.mp4
<vault>-80b60cf58788db69:2/28/2025, 10:53:29 AM->[ModuleFileHandler]  File <filename>.mp4 is not exist or already deleted on the database
<vault>-80b60cf58788db69:2/28/2025, 10:53:32 AM->[ModuleConflictResolver]  [conflict] Not conflicted or cancelled: livesync_log_2025-02-28.md
```
